jest.config.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
     },
     "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.jsx?$",
     "globals": {
-      "__baseUrl__": process.env.BROWSERSTACK_TEST_URL
+      "__baseUrl__": 'https://planet-webapp-51iccyhsu.vercel.app'
```

Comment:
Why have you removed this again now?

Reply:
@captainamiedi ...

@norbertschuler Maybe it makes sense to use a condition to pass different URLs when testing locally vs on browserstack?

Reply:
@sagararyal putting BROWSERSTACK_TEST_URL=... into your local .env file also lets you run the tests.